spec/models/author_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe Author, type: :model do
+  context 'database associations' do
+    it { is_expected.to have_many(:items) }
+  end
+
+  context 'model validations' do
+    it { is_expected.to validate_presence_of(:name) }
+
+    it { is_expected.to validate_presence_of(:service) }
+    it { is_expected.to validate_inclusion_of(:service).in_array(ServiceKind.list) }
+  end
+
+  context 'model attributes' do
+    it { is_expected.to have_db_column(:id).of_type(:integer).with_options(null: false) }
+
+    it { is_expected.to have_db_column(:service).of_type(:integer)
+                            .with_options(null: false) }
+
+    it { is_expected.to have_db_column(:profile_image_url).of_type(:string) }
+    it { is_expected.to have_db_column(:profile_url).of_type(:string) }
+    it { is_expected.to have_db_column(:name).of_type(:string)
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.